Index: launcher/launcher/launcher.cpp
===================================================================
--- launcher.orig/launcher/launcher.cpp	2010-09-01 11:46:33.000000000 +0200
+++ launcher/launcher/launcher.cpp	2010-09-01 11:47:56.000000000 +0200
@@ -19,12 +19,14 @@
 
 #include <QApplication>
 #include <QDeclarativeView>
+#include <QDeclarativeEngine>
 
 int main(int argc, char *argv[])
 {
     QApplication application(argc, argv);
 
     QDeclarativeView view;
+    view.engine()->setBaseUrl(QUrl::fromLocalFile("/usr/share/unity-qt/"));
     view.setSource(QUrl("./Launcher.qml"));
 
     return application.exec();
